Unity/Editor/InitVCRTForwarders.cs

```diff
@@ -32,18 +33,37 @@ static InitVCRTForwarders()
             return;
         }
 
+        List<string> searchFolders = new List<string>()
+        {
+            Application.dataPath,                                       // Assets folder
+            Path.GetFullPath(Path.Combine("Library", "PackageCache")),  // Library\PackageCache folder
+            Path.GetFullPath("Packages")                                // Pacakges folder

```

Comment:
Typo: `// Pacakges folder`
```suggestion
            Path.GetFullPath("Packages")                                // Packages folder
```